README.md

```diff
@@ -1003,6 +1003,9 @@ IEEE Trans. Pattern Anal. Machine Intell. (TPAMI), 2023. [PDF](https://arxiv.org
     - <a name="Wang21iccv"></a>Wang, Z., Ng, Y., Scheerlinck, C., Mahony., R.,  
 *[An Asynchronous Kalman Filter for Hybrid Event Cameras](https://openaccess.thecvf.com/content/ICCV2021/html/Wang_An_Asynchronous_Kalman_Filter_for_Hybrid_Event_Cameras_ICCV_2021_paper.html)*,  
 IEEE Int. Conf. Computer Vision (ICCV), 2021. [PDF](https://openaccess.thecvf.com/content/ICCV2021/papers/Wang_An_Asynchronous_Kalman_Filter_for_Hybrid_Event_Cameras_ICCV_2021_paper.pdf), [Code](https://github.com/ziweiWWANG/AKF), [YouTube](https://www.youtube.com/watch?v=XPz7laloKws), [Suppl](https://openaccess.thecvf.com/content/ICCV2021/supplemental/Wang_An_Asynchronous_Kalman_ICCV_2021_supplemental.pdf).
+- <a name="wangContiniuityCam"></a>Wang, Z., Hamann, F., Chaney, K., Jiang, W., Gallego, G., Daniilidis, K.,  
```

Comment:
capital W . See the comment below, about the key.

Reply:
I'm changing it to Wang23axiv. Are repeated entries okay? There is one existing entry for this key.

Reply:
No, I think there cannot be two anchors with the same name or the browser would not know where to jump to if the anchor was used in a link.

Reply:
Are there suggestions on how to avoid these repeated entries? Given the current tag format, it's very possible to have duplicated tags with very common last names.

Maybe numbering after the suggested format?

Reply:
arxiva arxivb, etc or similar, as it is done.